### PR TITLE
Remove install_yamls installation method for CRC

### PR DESCRIPTION
**Pull request by cjeanner**
As a pull request owner and reviewers, I checked that:
- [X] Appropriate testing is done and actually running
- [X] Appropriate documentation exists and/or is up-to-date:
  - [X] README in the role

### CODE REVIEW

**Review by cescgina — COMMENTED** [review]
/lgtm

File: ci_framework/roles/rhol_crc/tasks/main.yml [hunk 1]
```diff
@@ -63,33 +63,16 @@
...
 
-            - name: Setup RHOL/CRC
-              ansible.builtin.command: "{{ cifmw_rhol_crc_binary }} setup"
-              register: cifmw_rhol_crc_cmd_setup
-
-            - name: Start RHOL/CRC
-              ansible.builtin.command: "{{ cifmw_rhol_crc_binary }} start"
-              register: cifmw_rhol_crc_cmd_start
-
-    - name: Attach default network to CRC
```
Comment (rabi) [inline ci_framework/roles/rhol_crc/tasks/main.yml:84]: This has broken baremetal job.


+ cat
+ IP_ADDRESS_SUFFIX=6
+ CTLPLANE_IP_ADDRESS_SUFFIX=11
error: timed out waiting for the condition on nodenetworkconfigurationpolicies/enp6s0-crc-74q6p-master-0
gmake: *** [Makefile:1888: nncp] Error 1

Comment (gibizer) [reply ci_framework/roles/rhol_crc/tasks/main.yml:84]: Good catch. I see the same breakage https://github.com/openstack-k8s-operators/tcib/pull/102

Comment (hjensas) [reply ci_framework/roles/rhol_crc/tasks/main.yml:84]: Yes, see the same error here - https://github.com/openstack-k8s-operators/openstack-baremetal-operator/pull/106#issuecomment-1819814634

**Pull request merged**